docs/docs/getting-started/mini-constellation.md

```diff
@@ -0,0 +1,115 @@
+# Mini Constellation
+
+With `constellation mini`, users can quickly deploy and test Constellation without the need for a cloud subscription.
+
+The command uses virtualization to create a local cluster with one control-plane and one worker node.
+
+:::info
+
+Since mini Constellation is running on your local system, please note that common cloud features, such as load-balancing,
+attaching persistent storage, or auto-scaling, are unavailable.
+
+:::
+
+## Prerequisites
+
+* [Constellation CLI](./install.md#install-the-constellation-cli)
+* Linux operating system on amd64 hardware
+* At least 6GB RAM
+* A CPU with at least 4 cores
+* 20GB of free disk space
+* [KVM kernel module](https://www.linux-kvm.org/page/Main_Page) enabled
+  * Ensure your user is allowed to access `/dev/kvm` by running the following
+
+    ```shell
+    sudo chmod 666 /dev/kvm
+    ```
+
+* [Docker](https://docs.docker.com/engine/install/)
+* [xsltproc](http://xmlsoft.org/xslt/xsltproc.html)
```

Comment:
Is there an install guide available for this on the internet? Would link to that instead of the man page if there's a good one out there for common distros.